File: iniciante/python/1038-lanche.py
def lanche(codigo, unidade):
    precos = {1: 4.00, 2: 4.50, 3: 5.00, 4: 2.00, 5: 1.50}
    total = precos[codigo] * unidade
    return print(f'Total: R$ {total:.2f}')


def entrada():
    codigos = input().split(' ')
    prod = int(codigos[0])
    unidades = int(codigos[1])
    return prod, unidades


produto, quantidade = entrada()
lanche(produto, quantidade)

File: iniciante/c/1038-lanche.c
#include <stdio.h>

int main(){
    int codigo, quantidade;
    float total = 0;

    scanf("%d", &codigo);
    scanf("%d", &quantidade);

    if (codigo == 1){
        total = 4.0 * quantidade;
    }
    else if (codigo == 2){
        total = 4.5 * quantidade;
    }
    else if (codigo == 3){
        total = 5.0 * quantidade;
    }
    else if (codigo == 4){
        total = 2.0 * quantidade;
    }
    else if (codigo == 5){
        total = 1.5 * quantidade;
    }

    printf("Total: R$ %.2f\n", total);

    return 0;
}